app/components/common/navbar/__test__/Navbar.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import Slider from '../../slider';
+
+import Navbar, { DynamicHeroIcon } from '../index';
+
+describe('Navbar', () => {
+  it('renders', () => {
+    const { container } = render(<Navbar />);
+    expect(container).toBeInTheDocument();
+  });
+
+  it('renders the slider', () => {
+    const props = {
+      showSlider: true,
+      setShowSlider: jest.fn(),
+    };
+    const { container } = render(<Slider isOpen={props.showSlider} toggle={props.setShowSlider} />);
+    expect(container).toBeInTheDocument();
+  });
+
+  it('renders the dynamic hero icon', () => {
+    const props = {
+      name: 'randon',
```

Comment:
Minor spelling check

Reply:
It is just a placeholder text shouldn't make difference :)

Reply:
That's true, if you can try it by Monday
Lets do that else will merge Monday evening 👍 since it's noticed😅

Reply:
Addressed